.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/mirrors-yapf 
+  rev: 'v0.32.0'  # Use the sha / tag you want to point at
```

Comment:
This version is different from what is in the setup.py (0.40.1). And the repo is marked as deprecated: _use https://github.com/google/yapf directly now instead \o/_